nautobot/core/settings.yaml

```diff
@@ -0,0 +1,1659 @@
+          currently defaults to using `DatabaseFileStorage` to store such files directly in Nautobot's database;
+          however this is not typically the best option (see below) and may change in a future major release.
+
+      If your Nautobot server instance(s) and your Celery worker instance(s) share a common
+      [`MEDIA_ROOT`](#media_root) filesystem (as would typically be the case in a single-server installation of
+      Nautobot) then we recommend changing this to `"django.core.files.storage.FileSystemStorage"` to store
+      Job files on the filesystem (which will place them into a `files/` subdirectory under
+      [`MEDIA_ROOT`](#media_root)) instead of in the database.
+
+      If your Nautobot server instance(s) and Celery worker instance(s) do _not_ share a common filesystem,
+      we recommend using one of the [`django-storages`](https://django-storages.readthedocs.io/en/stable/) options
+      such as S3 to provide a storage backend that can be accessed by the server(s) and worker(s) alike.
+
+      !!! tip
+          For an example of using `django-storages` with AWS S3 buckets, visit the
+          [django-storages with S3](../guides/s3-django-storage.md) user-guide.
+
+      If you have neither a common `MEDIA_ROOT` filesystem nor an appropriate remote storage option, then
+      it's permissible to leave this at its default, but know that storing files in the database is provided here
+      as a "least-worst" option only.
+
+      !!! caution
+          It's typically safe to change this setting when initially updating to Nautobot 2.1.0 or later, as
+          there should be no pre-existing Job output files, although any existing scheduled Jobs that have
+          file _inputs_ may need to be deleted and recreated after doing so. However, once you've run any Jobs
+          that output to a file, changing storage backends will of course break any existing links to
+          Job output files in the previous storage backend.
+          Migrating Job stored files from one backend to another is out of scope for this document.
+    "environment_variable": "NAUTOBOT_JOB_FILE_IO_STORAGE"
+    "see_also":
+      "`STORAGE_BACKEND`": "#storage_backend"
+      "`JOB_CREATE_FILE_MAX_SIZE`": "#job_create_file_max_size"
+    "type": "string"
+    "version_added": "2.1.0"
+  "LOCATION_NAME_AS_NATURAL_KEY":
+    "default": false
+    "description": >-
+      `Location` names are not guaranteed globally-unique by Nautobot but in practice they often are.
+      Set this to `True` to use the location `name` alone as the natural key for `Location` objects.
+      Set this to `False` to use the sequence `(name, parent__name, parent__parent__name, ...)` as
+      the natural key instead.
+    "environment_variable": "NAUTOBOT_LOCATION_NAME_AS_NATURAL_KEY"
+    "is_constance_config": true
+    "type": "boolean"
+    "version_added": "2.0.0"
+  "LOGGING":
+    "additionalProperties": true
+    "default":
```

Comment:
This doesn't render that great:

![image](https://github.com/nautobot/nautobot/assets/75227981/cccc0e93-a532-4369-aa7e-a3092192db2f)

Reply:
See if it's a bit better now that I made some corrections... if it's still ugly, let's add a `default_literal` as appropriate.

Reply:
Still pretty messy:

![image](https://github.com/nautobot/nautobot/assets/5603551/e49258df-5336-4fc1-a0b2-ef515a80d6ad)

~As an alternative to `default_literal`, maybe we just should increase the `width` given to `pprint` in our rendering so that it doesn't need to wrap those `format` strings so aggressively?~ Unfortunately jinja2 `pprint` filter doesn't let you specify the width, so never mind.

Reply:
Using `default_literal`

![image](https://github.com/nautobot/nautobot/assets/75227981/1a16a2d9-0e9a-4ede-a72f-279604b2d6e4)